scripts/build_experimental_archive.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Define build and output directories
+gitTarget=${1:-"master"}
+buildDir=${2:-"./build"}
+
+# Create build directory and navigate into it
+mkdir -p "${buildDir}" && cd "${buildDir}"
+
+# Clone the repository
+git clone -v https://github.com/ooni/probe-cli.git
+
+# Navigate into the repository
+cd probe-cli
+
+git checkout master
```

Comment:
If you only want to clone each time, this line is redundant. Otherwise, I think it's fine to have it, but you would still need to add logic above to handle the case where the repository already exists.